src/main/resources/bidder-config/algorix.yaml

```diff
@@ -15,4 +15,4 @@ adapters:
       redirect-url:
       cookie-family-name: algorix
       type: redirect
-      support-cors: false
\ No newline at end of file
+      support-cors: false
```

Comment:
Redundant change, pls revert

Reply:
i'd say it's fine to have empty line at EOF. Let's keep this change :)